Code/GraphMol/SmilesParse/SmilesParseOps.cpp

```diff
@@ -151,6 +151,7 @@ void AddFragToMol(RWMol *mol, RWMol *frag, Bond::BondType bondOrder,
         newB->setOwningMol(mol);
         newB->setBeginAtomIdx(atomIdx1);
         newB->setEndAtomIdx(atomIdx2);
+        newB->setProp(RDKit::common_properties::_unspecifiedOrder, 1);
```

Comment:
@greglandrum interesting, is this related to #3307 by any chance?

Reply:
Unfortunately not. That seems to be connected to the aromaticity perception, not SMILES/SMARTS parsing